code/modules/mob/living/carbon/human/human.dm

```diff
@@ -50,6 +50,7 @@
 
 	internal_organs += new dna.species.mutanteyes()
 	internal_organs += new /obj/item/organ/brain
+	give_genitals()
```

Comment:
It checks it as a failsafe, but doing one check here instead of potentially doing it several times later is better.